components/dropdown/src/styles/bibStyles.scss

```diff
@@ -11,15 +11,32 @@
 
 /* stylelint-disable no-descending-specificity, selector-max-attribute */
 
-.container {
-  display: inline-block;
-  box-sizing: border-box;
-}
-
 :host {
   position: absolute;
   z-index: 1;
   display: none;
+
+  .container {
+    display: inline-block;
+    overflow: auto;
+    box-sizing: border-box;
+  }
+
+}
+
+:host([isfullscreen]) {
+ position: fixed;
+ top: 0;
+ left: 0;
+
+ .container {
+    top: 0;
+    left: 0;
+    width: 100dvw;
+    max-width: none;
+    height: 100dvh;
+    max-height: none;
+ }
 }
```

Comment:
For the second part in floatingUI.mjs, 
`position` function returns before calling `computePosition`, since  there is no need to event compute the position in fullscreen mode.